conda/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jina-hubble-sdk" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jina-hubble-sdk-{{ version }}.tar.gz
+  sha256: bab9b7b4d8c5965e099e07effd92cc0607b56eead0b4d26d6e5b473cb93daf8f
```

Comment:
Shall we convert this fixed sha256 to a template variable?

Reply:
> Shall we convert this fixed sha256 to a template variable?
No, it is  automatically created by running` python generate-conda-recipe.py`